From f2bccf76b978278209ea21f54aaa4d214fd0c26e Mon Sep 17 00:00:00 2001
From: juztamau5 <juztamau5@gmail.com>
Date: Fri, 11 Jun 2021 01:00:17 -0700
Subject: [PATCH 05/12] feat: Add supplyRatePerBlock() to Compound interface

---
 interfaces/compound/Token.sol | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/interfaces/compound/Token.sol b/interfaces/compound/Token.sol
index 417f1e9..431a8fe 100644
--- a/interfaces/compound/Token.sol
+++ b/interfaces/compound/Token.sol
@@ -18,4 +18,6 @@ interface cToken {
     function balanceOf(address _owner) external view returns (uint256);
 
     function underlying() external view returns (address);
+
+    function supplyRatePerBlock() external view returns (uint);
 }
-- 
2.34.1

